index.js

```diff
@@ -1,2 +1,4 @@
 'use strict';
-module.exports = str => encodeURIComponent(str).replace(/[!'()*]/g, x => `%${x.charCodeAt(0).toString(16).toUpperCase()}`);
+module.exports = function (str) {
+	return encodeURIComponent(str).replace(/[!'()*]/g, x => `%${x.charCodeAt(0).toString(16).toUpperCase()}`);
```

Comment:
`x => `%${x.charCodeAt(0).toString(16).toUpperCase()` I think this also should be converted to `function(...`